app/models/taxon_presenter.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
```

Comment:
I'm not managing to replicate this, but it might be that we're using different versions of node.